src/main/java/Duke/Duke.java

```diff
@@ -0,0 +1,74 @@
+package Duke;
+
+import Exceptions.InsufficientParametersException;
+import Exceptions.UnknownCommandException;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+
+public class Duke {
+
+    public static String handleInput(TaskManager taskManager, String[] input)
```

Comment:
`handleInput` seems a bit vague, maybe `executeUserCommand` or `executeUserInput` is clearer? Since you also made references to "commands" elsewhere in your code